prepare/tasks/response_assessment/pairwise_comparative_rating/single_turn.py

```diff
@@ -0,0 +1,21 @@
+from unitxt.blocks import Task
+from unitxt.catalog import add_to_catalog
+
+add_to_catalog(
+    Task(
+        inputs={
```

Comment:
In a recent PR, these are now called "input_fields", and "reference_fields" instead of "inputs" and "outputs". Again - we support for backward compatibility, but we should change all the internal code for this.